pkg/plugins/policies/meshhttproute/api/v1alpha1/helpers.go

```diff
@@ -5,7 +5,33 @@ type PolicyDefault struct {
 }
 
 func (x *To) GetDefault() interface{} {
+	var rules []Rule
+
+	for _, rule := range x.Rules {
+		var matches []Match
+
+		for _, match := range rule.Matches {
+			if match.Path == nil {
+				// According to Envoy docs, match must have precisely one of
```

Comment:
can we handle this in the XDS generation instead? It makes me somewhat nervous returning resources that the user didn't configure and wouldn't see match if they run `kubectl get`

Reply:
you are right

Reply:
addressed